Neos.Neos/Resources/Public/JavaScript/Components/TopBar/DropdownMenu.ts

```diff
@@ -0,0 +1,34 @@
+export default class DropDownMenu {
+  _root?: HTMLElement;
```

Comment:
Why don't you use `public` `protected` `private` to mark properties and methods instead of the underscore?
I would prefer that when TS already offers it.

Reply:
Yeah good point. Replaced it.